ingest/ingest_pipeline.py

```diff
@@ -140,20 +146,18 @@ def __init__(
         if ingest_cluster:
             self.cluster = self.initialize_file_connection(
                 "cluster", cluster_file)
-        if matrix_file is None:
-            self.matrix = matrix_file
         self.extra_log_params = {'study_id': self.study_id, 'duration': None}
         if subsample:
             self.cluster_file = cluster_file
             self.cell_metadata_file = cell_metadata_file
 
     @my_debug_logger()
+    # Will be replaced by MongoConnection
```

Comment:
The comment could use more detail.  When will it be replaced?  Why not now?  Is there a Jira ticket for that replacement

Reply:
a68804f